src/utilities/include/vec_view.h

```diff
@@ -46,16 +48,16 @@ class VecView {
   operator VecView<const T>() const { return {ptr_, size_}; }
 
   inline const T &operator[](size_t i) const {
-    if (i >= size_) {
-      throw std::out_of_range("Vec out of range");
-    }
+#if MANIFOLD_EXCEPTIONS
+    if (i >= size_) throw std::out_of_range("Vec out of range");
+#endif
     return ptr_[i];
   }
 
   inline T &operator[](size_t i) {
-    if (i >= size_) {
-      throw std::out_of_range("Vec out of range");
-    }
+#if MANIFOLD_EXCEPTIONS
+    if (i >= size_) throw std::out_of_range("Vec out of range");
+#endif
```

Comment:
Should we make another wrapper for these a bit like our `ASSERT`?

Reply:
probably, my goal for now is to get it to compile
I think we need better error handling